src/script/common/c_window_info.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+extern "C" {
+#include <lua.h>
+}
+
+#include "clientdynamicinfo.h"
+
+void push_window_info(lua_State *L, const ClientDynamicInfo &info);
```

Comment:
This is in its own file because I didn't want to introduce more code duplication, but wasn't sure where to put it